global-script.lua

```diff
@@ -304,6 +324,13 @@ function onLoad(saved_data)
     if saved_data ~= "" then
         local loaded_data = JSON.decode(saved_data)
         gameStarted = loaded_data.gameStarted
+        if loaded_data.playerBlocks then
```

Comment:
why are there conditionals around these objects get loaded?

Reply:
In my testing, there appeared to already be some save data but that save data does not have playerBlocks (and friends) initialized.  So this guards against the case of playerBlocks not (yet) being in the save data, in which case the initial values are used (by virtue of them not being overwritten here).

Reply:
ah yea i think i've run into that in the past, but i think once the data exists in the save file, then the checks could be removed?

Reply:
Data is now in the base save file, so removed the conditionals.